### PR TITLE
12.0 l10n il mja

### PR DESCRIPTION
**Pull request by mja-odoo**
Task:- https://www.odoo.com/web?#id=1966308&action=333&active_id=131&model=project.task&view_type=form&menu_id=4720

Description of the issue/feature this PR addresses:
Created new localization module for Israel Accounting
- Added COA
- Added Taxes

Pad:- https://pad.odoo.com/p/r.0b224fc1bfc6be352de70bec4380dfd3

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_il/data/account_data.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+	<data noupdate="1">
+
+		<!-- Account Tax Group -->
+		<record id="tax_group_vat_16" model="account.tax.group">
+			<field name="name">VAT 16%</field>
```
Comment (jva-odoo) [inline addons/l10n_il/data/account_data.xml]: @mja-odoo  add accounts applicability

File: addons/l10n_il/data/account_tax_template_data.xml [hunk 2]
```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+	<!-- Tax Tags, used in VAT report-->
+    <record id="tax_tag_il_00" model="account.account.tag">
```
Comment (jva-odoo) [inline addons/l10n_il/data/account_tax_template_data.xml]: @mja-odoo Please give clearer xml id in all tag.

File: addons/l10n_il/data/account_data.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record id="tax_group_vat_16" model="account.tax.group">
+        <field name="name">VAT 16%</field>
+    </record>
```
Comment (jco-odoo) [inline addons/l10n_il/data/account_data.xml]: @jcr-odoo Wikipedia (https://en.wikipedia.org/wiki/Taxation_in_Israel ) tells me the VAT is 17% and 16% is a really old one.  I also see there are no fiscal positions defined, while there is a clear distinction between export (Exempt) and the inland.

Comment (jcr-odoo) [reply addons/l10n_il/data/account_data.xml]: There is one tax that is 16%, its for palestina, partner approved wwith testing what is developed, so its good to go.

Comment (jco-odoo) [reply addons/l10n_il/data/account_data.xml]: Here we could also have fiscal positions to make it easier for the user.

**Comment by lathama** [comment]
@mja-odoo knowing that 12 is no longer updated, is this PR still needed?

**Comment by C3POdoo** [comment]
Dear @mja-odoo,

Thank you for your contribution but the version 12.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Comment by jva-odoo** [comment]
already moved to https://github.com/odoo/odoo/pull/38081

**Pull request closed without merging**